### PR TITLE
add remote compile documentation

### PR DESCRIPTION
**Pull request by Vivituhh**
## Introduced Changes

In this pull request the documentation is extended by remote compilation steps. 

Fixes #

## ToDo / Known Issues

None

## Ideas for Next Iterations (Not This PR)

None

## How to Test

You could try to set up remote compile for your machine.

### CODE REVIEW

File: docs/tooling/pepsi.md [hunk 1]
```diff
@@ -84,3 +84,18 @@ cargo install --path tools/pepsi
 ```
 
 and adding `~/.cargo/bin` to the `PATH`.
+
+## Remote Compile
+
+To use the remote compilation you need to create yourself an account on the rechenknecht. Therefore open an ssh connection to ```root@134.28.57.226```. There create a new account by ```adduser {name}``` and set a password with ```passwd {name}```, you will then be asked to type your password.
```
Comment (okiwi6) [inline docs/tooling/pepsi.md:90]: not `rechenknecht`, at the moment `remote-compiler`

File: docs/tooling/pepsi.md [hunk 2]
```diff
@@ -84,3 +84,18 @@ cargo install --path tools/pepsi
 ```
 
 and adding `~/.cargo/bin` to the `PATH`.
+
+## Remote Compile
+
+To use the remote compilation you need to create yourself an account on the rechenknecht. Therefore open an ssh connection to ```root@134.28.57.226```. There create a new account by ```adduser {name}``` and set a password with ```passwd {name}```, you will then be asked to type your password. 
+
+In the next step change to your user ssh connection ```{name}@134.28.57.226```. There clone the HULKs repository using https: ```https://github.com/HULKs/hulk.git```.
```
Comment (okiwi6) [inline docs/tooling/pepsi.md:92]: "Terminate the root ssh session and ..."

File: docs/tooling/pepsi.md [hunk 3]
```diff
@@ -84,3 +84,18 @@ cargo install --path tools/pepsi
...
+
+Back on your local machine do ```ssh-copy-id {name}@134.28.57.226```.
```
Comment (okiwi6) [inline docs/tooling/pepsi.md:94]: .. to copy your public ssh key to the remote-compiler

**Review by okiwi6 — CHANGES_REQUESTED** [review]
Thank you for providing helpful documentation, just some minor remarks

File: docs/tooling/pepsi.md (hunk 1 above)
Comment (Vivituhh) [reply docs/tooling/pepsi.md]: Done

File: docs/tooling/pepsi.md (hunk 2 above)
Comment (Vivituhh) [reply docs/tooling/pepsi.md]: Done

File: docs/tooling/pepsi.md (hunk 3 above)
Comment (Vivituhh) [reply docs/tooling/pepsi.md]: Done

**Review by okiwi6 — APPROVED** [review]
Thank you for the documentation :)

**Pull request merged**